src/ngMessages/messages.js

```diff
@@ -31,7 +31,9 @@ var jqLite = angular.element;
  *
  * ```html
  * <form name="myForm">
- *   <input type="text" ng-model="field" name="myField" required minlength="5" />
+ *   <label>Enter text:
```

Comment:
This is pretty basic form label stuff, in this kind of an example to me it makes sense to add label text.